corgi/collectors/brew.py

```diff
@@ -116,54 +116,72 @@ def _parse_remote_source_url(cls, url: str) -> str:
         path = parsed_url.path.removesuffix(".git")
         return f"{parsed_url.netloc}{path}"
 
+    @classmethod
+    def _bundled_or_golang(cls, component: str) -> Optional[str]:
+        # Process bundled deps only; account for typed golang deps of type:
+        # "golang(golang.org/x/crypto/acme)"
+        if component.startswith("bundled("):
+            c = component.removeprefix("bundled(")
+        elif component.startswith("golang("):
+            c = component
+        else:
+            return None
```

Comment:
Actually I see the only issue in tests is mypy. If you return "" here instead of None, I think you can avoid importing Optional. The "if not component: continue" logic below should continue to work, and mypy should be happier because the variable will always be of type str.
```suggestion
            return ""
````

Reply:
I need to adjust the return types here to keep MyPy happy now that I had a few bundled_components.append statements. I'm still making use of Optional but MyPy is no longer complaining.